src/components/GoogleAddressLookup/helpers/__test__/getSearchParams.spec.js

```diff
@@ -0,0 +1,61 @@
+import getSearchParams from '../getSearchParams';
+
+global.google = {
+    maps: {
+        LatLng: jest.fn(() => ({})),
+        LatLngBounds: jest.fn(() => ({})),
+    },
+};
+
+const searchOptions = [
+    {
+        bounds: {
+            sw: {
+                latitude: -63.941264,
+                longitude: 151.2042969,
+            },
+            ne: {
+                latitude: 63.941264,
+                longitude: -151.2042969,
+            },
+        },
+        types: ['address'],
+    },
+    {
+        location: {
+            latitude: -33.941264,
+            longitude: 151.2042969,
+        },
+        country: 'us',
+        radius: 150000,
+        types: ['address'],
+    },
+    {
+        country: ['us', 'ca'],
+    },
+];
+
+describe('getSearchParams', () => {
+    it('should convert searchOptions into google.maps.places.AutocompleteRequest search options', () => {
+        expect(getSearchParams(searchOptions[0])).toEqual({
+            bounds: global.google.maps.LatLngBounds(
+                global.google.maps.LatLng(-63.941264, 151.2042969),
+                global.google.maps.LatLng(63.941264, -151.2042969),
```

Comment:
as you mock this then it is returning an empty object so this is not right, so you need to mock this function to return something based on arg passed and then check that this values are equal to what it returned in the mock